cypress/support/commands/commands.ts

```diff
@@ -67,7 +76,9 @@ Cypress.Commands.add('iFrame', () => {
 const runTimestamp = +new Date();
 
 Cypress.Commands.add('createE2EResourceName', (context) => {
-  return cy.wrap(`e2e-test-${ runTimestamp }-${ context }`);
+  const randomStr = Math.random().toString(36).substr(2, 6);
```

Comment:
Cluster repository tests were calling `createE2EResourceName` in a `beforeEach` hook but I noticed that the same value was generated each time. I think this means that tests which use this method to name the resources they're attempting to create wont retry correctly, as they'll be trying to create resources with duplicate names